datafusion/sql/tests/cases/plan_to_sql.rs

```diff
@@ -495,11 +495,17 @@ fn test_table_references_in_plan_to_sql() {
         assert_eq!(format!("{}", sql), expected_sql)
     }
 
-    test("catalog.schema.table", "SELECT catalog.\"schema\".\"table\".id, catalog.\"schema\".\"table\".\"value\" FROM catalog.\"schema\".\"table\"");
-    test("schema.table", "SELECT \"schema\".\"table\".id, \"schema\".\"table\".\"value\" FROM \"schema\".\"table\"");
+    test(
```

Comment:
this is a driveby (unrelated) cleanup, right? (this is fine I am just verifying my understanding)

Reply:
It was necessary to get tests to pass, I assume it came from some other change in sqlparser, I didn't look too hard.

But it's not just cosmetic.

Reply:
this mostly changes `catalog` to `"catalog"`
a driveby is usage or raw literals (which is a nice thing on its own)